package.json

```diff
@@ -839,6 +841,12 @@
             "description": "%configuration.dotnet.backgroundAnalysis.compilerDiagnosticsScope%",
             "order": 30
           },
+          "dotnet.diagnostics.reportInformationAsHint": {
+            "type": "boolean",
+            "default": false,
```

Comment:
we should default this to on (report has hint), otherwise we're going to get a ton of complaints when we ship this.

Reply:
Should we prompt users on first launch to make a choice? I think if we ship it on then no one will ever find it. Users who expect info diagnostics will continue to think our extension is broken and may never check GH.

Reply:
Changed the default to match the current behavior. We can look at prompting the user in the future.